src/Admin/AdminInterface.php

```diff
@@ -32,6 +32,7 @@
  *
  * @method array                           configureActionButtons(string $action, ?object $object = null)
  * @method string                          getSearchResultLink(object $object)
+ * @method array                           getDefaultFilterParameters()                                    // NEXT_MAJOR: Remove this
```

Comment:
IMHO you can add a general NEXT_MAJOR comment above all `@method`s

Reply:
Edit: There is already one. So you can remove this information

Reply:
@VincentLanglet @phansys what do you think?=)

Reply:
I agree with this https://github.com/sonata-project/SonataAdminBundle/pull/6700#discussion_r544212529:
https://github.com/sonata-project/SonataAdminBundle/blob/ab4c3c4761e1b842cb2dde7b61db38c2c6cd20f5/src/Admin/AdminInterface.php#L31

Reply:
You review too long.. When im create PR there was other methods martked same comment. Guys(